elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManagerFactory.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.impl;
+
+import org.hibernate.search.engine.impl.DefaultIndexManagerFactory;
+import org.hibernate.search.indexes.spi.IndexManager;
+
+/**
+ * Index manager capable of resolving the "elasticsearch" alias. All other names/aliases will be handled by the default
+ * implementation.
+ * <p>
+ * Registered through the Java service loader mechanism.
+ *
+ * @author Gunnar Morling
+ */
+public class ElasticsearchIndexManagerFactory extends DefaultIndexManagerFactory {
```

Comment:
@gustavonalle you might want to keep this problem in mind too ^

https://hibernate.atlassian.net/browse/HSEARCH-2115